patches/net/minecraft/world/World.java.patch

```diff
@@ -92,19 +93,29 @@
                  {
                      try
                      {
-@@ -1363,6 +1382,7 @@
+@@ -1363,6 +1383,7 @@
                      this.getChunk(tileentity.getPos()).removeTileEntity(tileentity.getPos());
                  }
              }
 +            CarpetProfiler.end_current_entity_section();
          }
  
          this.processingLoadedTiles = false;
-@@ -1393,6 +1413,7 @@
+@@ -1393,6 +1414,7 @@
  
              this.addedTileEntityList.clear();
          }
 +        CarpetProfiler.end_current_section();
```

Comment:
Where is the start_section for this end_section statement?

Reply:
Isn't it already there?
[Start Section For Line 108](https://github.com/DeadlyMC/carpetmod/blob/UnloadedEntity/patches/net/minecraft/world/World.java.patch#L73-L73)
AND
[Start Section For Line 100](https://github.com/DeadlyMC/carpetmod/blob/UnloadedEntity/patches/net/minecraft/world/World.java.patch#L83)

Reply:
ok, read the +- wrong - thought for some reason you added these, but these were there before. np.